src/VirtualDom/Styled.elm

```diff
@@ -250,6 +257,139 @@ lazyHelp7 fn arg1 arg2 arg3 arg4 arg5 arg6 arg7 =
         |> toUnstyled
 
 
+type Scope
+    = Scope String
+
+
+encodeScope : Scope -> Json.Encode.Value
+encodeScope (Scope scope) =
+    Json.Encode.string scope
+
+
+uncurry : (a -> b -> c) -> ( a, b ) -> c
```

Comment:
`uncurry` and `mapOnto` don't feel to me like they belong in this file 🤔 Possibly worth a new file? Although maybe Richard is cool with these being in here..?

Reply:
`uncurry` is gone now and `mapOnto` is used only in VirtualDom/Styled.elm.  Any better suggestions for where it should go?  Maybe a new List.Extra file?

Reply:
I might inline its uses, rather than having a helper? But I don't feel strongly either way